src/index.html

```diff
@@ -2,15 +2,51 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="get"
+      class="search__bar search__bar--big"
+      data-qa="big">
+      <label class="search__bar-label">
+        <img
+          src="./images/Search.svg"
+          alt="search"
+          class="search__bar-img">
+        <input
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+          class="search__bar-input search__bar-input--big">
+      </label>
+    </form>
+
+
+    <form
+      action="#"
+      method="get"
+      class="search__bar search__bar--small"
+      data-qa="small">
+      <label class="search__bar-label">
+        <img
+          src="./images/Search.svg"
+          alt="search"
+          class="search__bar-img search__bar-img--small">
+        <input
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+          class="search__bar-input search__bar-input--small">
+      </label>
+    </form>
+
   </body>
+
 </html>
```

Comment:
```suggestion
      </label>
    </form>
  </body>
</html>
```
Don't leave empty lines between parent and child elements